carts/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Carts app url mapping
+"""
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("", views.CartsAPIView.as_view(), name="Cart Api"),
```

Comment:
Let's use snake case for all the API names here and all other places please.
```suggestion
    path("", views.CartsAPIView.as_view(), name="cart_api"),
```

Reply:
Done